cmake/sDiffConfig.cmake.in

```diff
@@ -0,0 +1,8 @@
+set(sDiff_VERSION @sDiff_VERSION@)
+
+find_package(Eigen3 REQUIRED)
```

Comment:
You can use `find_dependency` here, see  https://cmake.org/cmake/help/v3.5/module/CMakeFindDependencyMacro.html .